### PR TITLE
ENG-19798-update-readme-and-build-v-add-sample

### PR DESCRIPTION
**Pull request by alexaltman**
- update build version to 0.1.15
- add AddressBookClient section to README 
- add SampleAddressBookClient
- update docs

### CODE REVIEW

**Review by yangAtSpoken — APPROVED** [review]

**Review by yangAtSpoken — APPROVED** [review]

File: NodeJS/README.md [hunk 1]
```diff
@@ -88,6 +88,34 @@ The AUXCodeClient retrieves sub account aux codes, effective aux code given sub
...
     let appIdAuxCodes = await auxCodeClient.getAUXCodesForEffectiveAppId();
 
+## AddressBookClient
+
+The AddressBookClient could be used to get sub-account Address Book with all active contacts.
+With search endpoint contacts could be fetched by multiple optional params: type (one or many of Agent/Supervisor/Outbound/ACO/VDN separated by comma); partial name/destination; full virtualExtension.
+The search request could contain pagination (pageSize; page number) and sorting (by name/type/virtualExtension/destination; direction ASC/DESC) params.
+
+    // create an instance of AddressBookClient
+    let addressBookClient = createAddressBookClient(endpoint, apiKey)
```
Comment (joe-s-avaya) [inline NodeJS/README.md]:
```suggestion
    const addressBookClient = createAddressBookClient(endpoint, apiKey)
```

And you need to put code [in code blocks](https://commonmark.org/help/tutorial/09-code.html)

Comment (joe-s-avaya) [reply NodeJS/README.md]: also, no need to indent

File: NodeJS/README.md [hunk 2]
```diff
@@ -113,6 +141,12 @@ AUX Codes (or Auxiliary Codes) are used to track the time an agent has deliberat
 
 `node lib/sample/SampleAUXCodeClient.js --endpoint='http://localhost:8081' --api_key=YOUR-API-KEY`
 
+## Run SampleAddressBookClient
+
+Address Book contains contacts of different types Agent/Supervisor/Outbound/ACO/VDN. This sample client fetches full AddressBook and sends search request.
```
Comment (joe-s-avaya) [inline NodeJS/README.md]:
```suggestion
An Address Book contains contacts of the following types:
Agent | Supervisor | Outbound | ACO | VDN

This sample client can fetch a full Address Book or execute queries on an AddressBook.
```

**Review by enrique-prado — APPROVED** [review]

**Review by joe-s-avaya — APPROVED** [review]

**Comment by alexaltman** [comment]
thanks all

**Pull request merged**